packages/solo/src/start.js

```diff
@@ -98,7 +98,7 @@ async function buildSwingset(
   const pluginDir = path.resolve('./plugins');
   fs.mkdirSync(pluginDir, { recursive: true });
   const pluginsPrefix = `${pluginDir}${path.sep}`;
-  const pluginRequire = mod => {
+  const pluginRequire = async mod => {
```

Comment:
Should this be renamed to `pluginImport` ? I think one of the unit tests made the same change.